modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/form_instance_records_search_container.jsp

```diff
@@ -123,6 +123,23 @@ PortletURL portletURL = ddmFormViewFormInstanceRecordsDisplayContext.getPortletU
 							%>
 
 						</c:when>
+						<c:when test="<%= StringUtil.equals(ddmFormField.getType(), DDMFormFieldTypeConstants.CHECKBOX) %>">
+							<c:choose>
+								<c:when test='<%= StringUtil.equals(ddmFormViewFormInstanceRecordsDisplayContext.getColumnValue(ddmFormFieldsMap.get(ddmFormField.getName()), StringPool.BLANK, ddmFormFieldValuesMap.get(ddmFormField.getName())), "Yes") %>'>
```

Comment:
@mlan0, the word `Yes` shouldn't be replaced by `True`?

Will this condition (`<c:when ...`) keep working as intended even for form entries submitted on languages different from English?

Reply:
Hi @natocesarrego , It shouldn't be replaced because the two values are "Yes" and "No". But this is not working for different languages. I will fix it!

Reply:
Ok. Please resend.